.eslintrc.json

```diff
@@ -17,7 +17,8 @@
     "jsx-a11y/label-has-associated-control": 0,
     "jsx-a11y/no-static-element-interactions": 0,
     "compilerOptions": { "target": "ES6", "module": "commonjs" },
-    "max-len": ["error", { "code": 100, "ignoreStrings": true, "ignoreTemplateLiterals": true }]
+    "max-len": ["error", { "code": 100, "ignoreStrings": true, "ignoreTemplateLiterals": true }],
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": true}]
```

Comment:
Should this be here? It only disables the `max-length` of the devDependencies. You should set disable `import/noextranous` instead

Reply:
Correct me if am wrong but line 21 `import/no-extraneous-dependencies` does something similar to `import/noextranou`. line 20, on the other hand, does what you are talking about above. If you look closely, line 20 shows it was a change made by me but it is not.

Reply:
It's good now, I had seen a different line structure instead. let me approve